rpcs3/Emu/RSX/GL/GLOverlays.h

```diff
@@ -122,22 +178,37 @@ namespace gl
 
 				if (scissor_enabled) glDisable(GL_SCISSOR_TEST);
 				if (cull_face_enabled) glDisable(GL_CULL_FACE);
-				if (blend_enabled) glDisable(GL_BLEND);
 				if (stencil_test_enabled) glDisable(GL_STENCIL_TEST);
 
+				if (use_blending)
+				{
+					if (!blend_enabled)
+						glEnable(GL_BLEND);
+
+					glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
```

Comment:
I think a call to set the blend equation is missing here:
glBlendEquation(GL_FUNC_ADD);

Fixes the black screen in Silent Hill 2 for me.

Reply:
Yea, its missing the eqn